moonwalk/blocks/lnd.py

```diff
@@ -0,0 +1,116 @@
+from decimal import Decimal as D
+
+from hexbytes.main import HexBytes
+from .eth import EthereumProxy
+from eth_abi.abi import decode_abi
+from eth_account.account import Account
+from eth_utils.address import to_checksum_address, is_address
+from eth_hash.auto import keccak
+
+from moonwalk import settings
+
+
+DECIMALS = pow(10, 18)
+
+
+class LendingblockProxy(EthereumProxy):
+
```

Comment:
do we need to override all these methods for LND?

Reply:
Yes, most of the functionality for `LND` is similar to `ETH` (eg. `get_gas_price`, `send_single_transaction`) and that's why in `LendingblockProxy` class we override only necessary methods (eg. `send_money`).